### PR TITLE
Transalated to english, almost!

### PR DESCRIPTION
**Pull request by yourfrienddhruv**
Tried to translate for larger audience, world needs to know about this!
 Kindly pardon any silly mistakes.

Used Bing! Translate (the only good translator ) -> Manually edited for grammar and tone.

### CODE REVIEW

File: indexr_white_paper/indexr_white_paper.en.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+INDEXR: Real-time, Hadoop based Data Warehouse
```
Comment (flowbehappy) [inline indexr_white_paper/indexr_white_paper.en.md]: We usually use "IndexR" not "INDEXR", the same as others following.

File: indexr_white_paper/indexr_white_paper.en.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+-------------|----------|
+' Date ' | int |
+' Contry ' | string |
+' Campaign_id ' | Long |
+' Impressions ' | Long |
+' Clicks ' | Long |
+
+The data file is called segment, and a segment saves a portion of a table, containing all the columns, as shown below.
+
+! [Segment_file] (segment_file. png)
```
Comment (flowbehappy) [inline indexr_white_paper/indexr_white_paper.en.md]: `Segment_file`

File: indexr_white_paper/indexr_white_paper.en.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+* **Use Direct memory (directly Memory)**. INDEXR is primarily written using JAVA8, while Java's heap memory (HEAP) and garbage collection (GC) patterns face larger challenges in large data operations scenarios. When large memory is required (over 32G) and data is updated frequently, the problem of the JVM's GC is more obvious, which is prone to performance instability, and the memory model of the object instance is often wasteful. In the INDEXR project, we store all the stored data and operation temporary data outside the heap, and manually manage the memory request release. This improves code complexity, but saves more than 1/2 memory from the traditional heap memory pattern, and without the GC cost, an assignment operation involving large amounts of data can typically use memory copies to save a lot of CPU cycles.
```
Comment (flowbehappy) [inline indexr_white_paper/indexr_white_paper.en.md]: ~(directly Memory)~

File: indexr_white_paper/indexr_white_paper.en.md (hunk 2 above)
Comment (flowbehappy) [reply indexr_white_paper/indexr_white_paper.en.md]: So does the other images.

**Review by flowbehappy — CHANGES_REQUESTED** [review]

**Comment by flowbehappy** [comment]
@yourfrienddhruv Think you for your contribution! That is really helpful. 
I have added some comments before it can be merged.

**Comment by yourfrienddhruv** [comment]
Fixed as suggested.

**Pull request merged**